pkg/plugin/ws_data_proxy.go

```diff
@@ -60,24 +60,10 @@ func (wsdp *wsDataProxy) readMessage() {
 			_, message, err := wsdp.wsConn.ReadMessage()
 
 			if err != nil {
-				if websocket.IsCloseError(err, websocket.CloseGoingAway) {
-					log.DefaultLogger.Error("Disconnection Error", "error", err.Error())
-					sendErrorFrame(fmt.Sprintf("%s: %s", "Disconnection Error", err.Error()), wsdp.sender)
-
-					wsdp.wsConn, err = wsdp.wsConnect()
-					if err != nil {
```

Comment:
I thought that we had removed that reconnection logic already, I like the idea of removing it from here and letting Grafana restart the stream and them we try to reconnect 👍